pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -102,16 +98,17 @@ let
       openssl
       (libxml2.override {enableHttp = true;})
       icu
+      libuuid
     ]
       ++ lib.optionals (olderThan "13") [ libxcrypt ]
       ++ lib.optionals jitSupport [ llvmPackages.llvm ]
       ++ lib.optionals lz4Enabled [ lz4 ]
       ++ lib.optionals zstdEnabled [ zstd ]
-      ++ lib.optionals systemdSupport' [ systemdLibs ]
+      ++ lib.optionals systemdSupport [ systemdLibs ]
       ++ lib.optionals pythonSupport [ python3 ]
       ++ lib.optionals gssSupport [ libkrb5 ]
       ++ lib.optionals stdenv'.hostPlatform.isLinux [ linux-pam ]
-      ++ lib.optionals (!stdenv'.hostPlatform.isDarwin) [ libossp_uuid ];
+      ++ lib.optionals (!stdenv'.hostPlatform.isDarwin) [ e2fsprogs ];
```

Comment:
Can you explain why both `libossp_uuid` & `e2fsprogs` are needed? From what I've read in the commit message, only one of them is required?

Reply:
> libossp_uuid is removed, but you mean libuuid (further up) and e2fsprogs, I guess.

Correct, good morning ma27 🙃